.github/workflows/setup-helper.yml

```
@@ -0,0 +1,93 @@
+        type: choice
+        options:
+          - development
+          - test
+          - production
+
+jobs:
+  generate-env:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)